aws/cft.yml

```diff
@@ -0,0 +1,141 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Conjur CE Stack, backed by a PostgreSQL database on RDS
+
+Parameters:
+  ConjurAccount:
+    Type: String
+    Description: Name of Conjur account
+
+  ConjurAdminPassword:
+    Type: String
+    NoEcho: true
+    Description: Password for Conjur admin account
+
+  DBAdminUser:
+    Type: String
+    Description: RDS instance admin user
+    Default: conjuradmin
+
+  DBAdminPassword:
+    Type: String
+    NoEcho: true
+    Description: RDS instance admin password
+
+  ImageId:
+    Type: AWS::EC2::Image::Id
+    Description: Conjur CE AMI
+
+  KeyName:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: AWS keypair to use
+
+  VpcId:
+    Type: AWS::EC2::VPC::Id 
+    Description: VPC in which to launch Conjur instances
+
+  VpcSubnetIds:
+    Type: "List<AWS::EC2::Subnet::Id>"
+    Description: Subnets in which to launch Conjur and Postgres. Specify two, in different Availability Zones.
```

Comment:
There are neat ways to provide drop-down choices here, but we can do that in another issue/PR.

Reply:
I get a list of subnets in a drop-down here. Do you not?